src/basic/os-util.h

```diff
@@ -3,32 +3,41 @@
 
 #include <stdio.h>
 
+typedef enum ImageClass {
+        IMAGE_MACHINE,
+        IMAGE_PORTABLE,
+        IMAGE_EXTENSION,
+        IMAGE_SYSCFG,
+        _IMAGE_CLASS_MAX,
+        _IMAGE_CLASS_INVALID = -EINVAL,
+} ImageClass;
```

Comment:
why move this over?

if you move this over image_class_to_string() + image_class_from_string() should be moved as well. 

bit i don't see why we'd move this over.

Reply:
Because I am told that I'm not allowed to add full file paths in include so I can't do something like #include "src/shared/discover-image.h"